server/liveblog/items/items.py

```diff
@@ -50,6 +50,16 @@ class ItemsResource(ArchiveResource):
         'item_type': {
             'type': 'string'
         },
+        'name': {
+            'type': 'string',
+            'minlength': 1,
+            'maxlength': 30
+        },
+        'content': {
```

Comment:
the `content` has the same role as text but here we have `maxlength` and `minlength` constraints